lib/fastlane/plugin/android_sdk_update/actions/android_sdk_update_action.rb

```diff
@@ -52,6 +68,15 @@ def self.run(params)
         end
       end
 
+      def self.download_and_extract_sdk(download_url, sdk_path)
+        FastlaneCore::CommandExecutor.execute(
+          command: "wget -O /tmp/android-sdk-tools.zip #{download_url}",
+          print_all: true, print_command: true)
+        FastlaneCore::CommandExecutor.execute(
+          command: "unzip /tmp/android-sdk-tools.zip -d #{sdk_path}",
+          print_all: true, print_command: true)
+      end
```

Comment:
We need to link "adb" and maybe "aapt" to /usr/local/bin

Because otherwise certain fastlane actions would not work:

- adb (https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/adb.rb)
- screengrab (https://github.com/fastlane/fastlane/blob/master/screengrab/lib/screengrab/android_environment.rb)

Reply:
Yes -- this is an odd behaviour in screengrab: it figures out where the android sdk lives, and then completely ignores that path when calling adb!

Instead of creating files in /usr/local/bin, I propose we could add our android-sdk/platform-tools directory to the PATH variable. What do you think?

Reply:
I can think of a few usecases, where this might cause some problems:

If you just execute Screengrab `fastlane screengrab`
There is no point for us, to modify the PATH var.